### DISCUSSION TITLE
Drag and drop sorting of tags

### DISCUSSION THREAD

**Discussion by seabasss** [orchidjs/tom-select#456] [Ideas]
I love almost everything about tom select, but one thing that I'm missing is drag and drop sorting of the tags.

Tagify has this feature which is really cool:
https://codepen.io/vsync/pen/jOqYOVJ

Thanks for taking the request into consideration!

**Comment by ghost** [comment]
View my quick&dirty example: https://github.com/orchidjs/tom-select/discussions/576
